### ISSUE TITLE
`simd_div` and `simd_mod` ICEs for non-power-of-two length vectors on Aarch64

### ISSUE THREAD

**Issue by calebzulawski** [rust-lang/portable-simd#427]
For non-power-of-two length vectors on Aarch64 (#422):
* `simd_div` fails when the rhs argument is 0
* `simd_mod` fails when the rhs argument is MIN
These still need reproducers to submit to LLVM

**Comment by workingjubilee** [comment]
oh cool.

**Comment by workingjubilee** [comment]
@calebzulawski was this before we made our weird layout changes or just always

**Comment by workingjubilee** [comment]
also on what element types?

**Comment by calebzulawski** [comment]
Yeah, only with `#[repr(simd, packed)]`.  I want to say it was for i8 but I don't remember for sure.
